rsconnect/main.py

```diff
@@ -1979,6 +2006,11 @@ def _write_framework_manifest(
     with cli_feedback("Inspecting Python environment"):
         _, environment = get_python_env_info(directory, python, force_generate)
 
+    if app_mode == AppModes.PYTHON_SHINY:
+        with cli_feedback("Inspecting Shiny for Python app"):
+            if is_express_app(entrypoint + ".py", directory):
+                entrypoint = "shiny.express.app:" + escape_to_var_name(entrypoint + ".py")
+
```

Comment:
This is the part that adds support for Shiny Express when writing manifest files.